schedule/yast/maintenance/mau_extratest_yast-cmd_SLE12.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: mau-extratests-yast-cmd
+description:    >
+  Testsuite maintained at https://gitlab.suse.de/qa-maintenance/qam-openqa-yml. Run
+  console tests against aggregated test repo.
```

Comment:
I guess for the decription we could leave that sentence "Testsuite ... openqa-yml' so we can track all those test suite, but the description we should improve it, we also now have the same description for the both test suites, wdyt?

Reply:
modified